File: 프로그래머스/unrated/181880. 1로 만들기/1로 만들기.js
function solution(num_list) {
    let count = 0;
		for ( let num of num_list ) {
			while ( num != 1 ) {
				count++;
				num = ( num % 2 == 0 ) ? num / 2 : ( num - 1 ) / 2;
			}
		}
		return count;
}

File: 프로그래머스/unrated/181880. 1로 만들기/1로 만들기.java
class Solution {
    public int solution(int[] num_list) {
        int count = 0;
		for ( int num : num_list ) {
			while ( num != 1 ) {
				if ( num % 2 == 0 ) {
					count++;
					num = num / 2;
				} else if ( num % 2 == 1 ) {
					count++;
					num = ( num - 1 ) / 2;
				}
			}
		}
		return count;
    }
}